src/mps/mps.jl

```diff
@@ -772,15 +772,17 @@ function expect(psi::MPS, ops; kwargs...)
   Ns = length(site_range)
   start_site = first(site_range)
 
+  el_types = map(o -> ishermitian(op(o, s[start_site])) ? Float64 : ElT, ops)
```

Comment:
My guess is that this should be `real(ElT)` instead of `Float64`, for example in case someone is using single precision.

Reply:
Thanks for catching that